PYME/ui/AUIFrame.py

```diff
@@ -61,7 +64,7 @@ def AddPage(self, page=None, select=True,caption='Dummy', update=True):
             self._mgr.Update()
             pn = self._mgr.GetPaneByName(self.pane0)
             if pn.IsNotebookPage():
-                print((pn.notebook_id))
+                logger.debug((pn.notebook_id))
```

Comment:
```suggestion
                #logger.debug((pn.notebook_id))
```
I think this one can just be commented out (potentially also the creating fold panel one below, although that probably doesn't hurt to log). In general, when we are replacing print statements with logging calls, we should check if they still serve a purpose and if the provide enough context to be useful (just printing the value of a handle clearly doesn't). If they don't have enough context, then I think it's best to just comment them out.